dist/prototypes.py

```diff
@@ -39,7 +39,8 @@
 
 """
 
-if not [f for f in filter_if_fast(source_files(), prefix="../")]:
+files = [f for f in source_files()]
+if not [f for f in filter_if_fast(files + ["../dist/prototypes.py"], prefix="../")]:
```

Comment:
Would it exit earlier, if no file changes were made, and only prototypes.py was changed?

Reply:
If that's the case, doesn't it hold true for all the s_all files?

Reply:
Good point.. I think this needs fixing for all files. I'll spin up a new ticket for this